tests/test_split_transform.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) MONAI Consortium
```

Comment:
When we finalized the transform name, could you please help change the file name to `test_XXX` without `_transform` to be consistent with other tests?

Thanks.